packages/frontend/admin/src/modules/accounts/components/user-form.tsx

```diff
@@ -121,11 +137,7 @@ function UserForm({
             <div key={feature}>
               <ToggleItem
                 name={feature}
-                checked={(
-                  changes.features ??
-                  defaultValue?.features ??
-                  []
-                ).includes(feature)}
+                checked={changes.features?.includes(feature) ?? false}
```

Comment:
use `defaultChecked`?

Reply:
Now it works fine. If I use the `defaultChecked` attribute, it also needs to handle the logic of clearing after exiting.